src/integrations/watchers/addon-update-watcher.php

```diff
@@ -37,15 +37,8 @@ class Addon_Update_Watcher implements Integration_Interface {
 	 * Registers the hooks.
 	 */
 	public function register_hooks() {
-		\add_action(
-			'update_site_option_auto_update_plugins',
-			[
-				$this,
-				'toggle_auto_updates_for_add_ons',
-			],
-			10,
-			3
-		);
+		\add_action( 'add_option_auto_update_plugins', [ $this, 'call_toggle_auto_updates_with_empty_array' ], 10, 2 );
```

Comment:
Is there a reason that you didn't used `add_site_option_auto_update_plugins`. I would say that would be a better action to call because of multisite installs.

I made a commit suggestion for this.
```suggestion
		\add_action( 'add_site_option_auto_update_plugins', [ $this, 'call_toggle_auto_updates_with_empty_array' ], 10, 2 );
```

Reply:
Great suggestion @andizer.